README.md

```diff
@@ -29,9 +29,11 @@ NB: a trailing slash is required.
 ## Development
 
 ### Linting
+
 Titan support linting according to the official [Flutter static analysis options](https://dart.dev/guides/language/analysis-options).
 
 The linter can be launched using:
+
 ```
 dart format .
 flutter analyze --fix
```

Comment:
It is `dart fix --apply`and we should apply it before the formating to ensure formatting is correct event after applying changes

Reply:
We can also run `dart fix --dry-run` to visualize the change that would be done if the run was with `--apply`